src/main/java/org/broadinstitute/hellbender/engine/spark/GATKRegistrator.java

```diff
@@ -52,6 +56,9 @@ public void registerClasses(Kryo kryo) {
         //register to avoid writing the full name of this class over and over
         kryo.register(PairedEnds.class, new FieldSerializer<>(kryo, PairedEnds.class));
 
+        kryo.register(Allele.class, new FieldSerializer<>(kryo, Allele.class));
+        kryo.register(FastGenotype.class, new FieldSerializer<>(kryo, FastGenotype.class));
+        kryo.register(VariantContext.class, new FieldSerializer<>(kryo, VariantContext.class));
```

Comment:
If these 3 registrations are not strictly necessary, can we drop them? According to Kryo docs, `FieldSerializer` is already the default.